pkg/client/listers_generated/cluster/internalversion/machinedeployment.go

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Shouldn't these also be gone?

Reply:
are you referring to pkg/client/listers_generated/cluster/internalversion/... should be gone or just pkg/client/listers_generated/cluster/internalversion/machinedeployment.go

Reply:
anything with internalversion shouldn't exist

Reply:
deleted the internalversion dir